src/rest/client.ts

```diff
@@ -56,14 +56,7 @@ export class RestClient {
       method,
       body,
     });
-    this.assertOk(response, { method, url });
 
     return response.json();
   }
-
-  private assertOk(response: Response, opts: { method: string; url: string }) {
```

Comment:
Some times we want to test for 400 type errors, and this was just logging.

Tests should assert on responses so we don't need a built in assertion like this anyway